src/shims/unix/fd.rs

```diff
@@ -310,6 +381,41 @@ impl FdTable {
 
 impl<'tcx> EvalContextExt<'tcx> for crate::MiriInterpCx<'tcx> {}
 pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
+    /// For a specific unique file descriptor id, get its ready events and update
+    /// the corresponding ready list.
```

Comment:
Do I understand correctly that it is always correct to call this "unnecessarily", i.e., when nothing actually changed in readiness? eventfd seems to rely on that, so this may be good to document.